compiler/noirc_driver/src/stdlib.rs

```diff
@@ -0,0 +1,24 @@
+use rust_embed::RustEmbed;
+
+#[derive(RustEmbed)]
+#[folder = "../../noir_stdlib/src"]
+#[cfg_attr(not(target_os = "windows"), prefix = "std/")]
```

Comment:
Need to double check if this windows shenanigans is still needed -- I removed most of the windows specific config code that was in file_reader

Reply:
It should still be necessary, although we're not supporting Windows officially anymore it's not a particularly burdensome bit of code.